### ISSUE TITLE
news36tech Anti-adblock

### ISSUE THREAD

**Issue by arshadbarves** [uBlockOrigin/uAssets#22634]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://news36tech.com/best-loan-providers-for-legal-professionals-funding-justice/
```

### Description

Filter doesn't work 

### Other extensions used

No

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="1792" alt="image" src="https://github.com/uBlockOrigin/uAssets/assets/39230484/94ebd4b3-fc01-4d56-8398-913371fcd32b">

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Chromium: 122
filterset (summary):
 network: 125175
 cosmetic: 45587
 scriptlet: 20299
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  easylist: 81745-12, 11m Δ
  easyprivacy: 50270-6, 11m Δ
  plowe-0: 3782-1184, 8d.22h.8m
  ublock-badware: 8001-142, 11m Δ
  ublock-filters: 37348-362, 11m Δ
  ublock-privacy: 744-6, 11m Δ
  ublock-quick-fixes: 124-5, 11m Δ
  ublock-unbreak: 2223-34, 11m Δ
  urlhaus-1: 8725-0, 12h.1m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1940 ms (selfie)
 maxAssetCacheWait: 137 ms
 cacheBackend: browser.storage.local
```
</details>

**Comment by uBlock-user** [comment]
`news36tech.com##+js(rmnt, script, fakeAd.offsetHeight)` works for you ?

**Comment by uBlock-user** [comment]
@ghajini `news36tech.com##+js(no-fetch-if, adsbygoogle)` works, link is generated

**Comment by uBlock-user** [comment]
> then link is hidden

Yes after 3 steps, at the final page, link is not getting generated, but I don't see any anti-adb

**Comment by uBlock-user** [comment]
You mean remove the exception - `@@||doubleclick.net^$script,xhr,domain=theringtonesworld.in|jobkijankari.in|newsloti.com|newzwala.co.in|iplquotes.com|kaisekareblog.com|w3hindi.in|jobwaala.in|nhmgujarat.in|minijankari.com` ? I don't know if nobody knows the filter for that exception, then perhaps we need to see how AdGuard folks deal with gplinks cases

**Comment by ItsProfessional** [comment]
For gplinks domains, we have a specific section in `ubo-link-shorteners.txt`.
